x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elastic/ElasticInferenceServiceSettings.java

```diff
@@ -17,6 +18,8 @@ public class ElasticInferenceServiceSettings {
     @Deprecated
     static final Setting<String> EIS_GATEWAY_URL = Setting.simpleString("xpack.inference.eis.gateway.url", Setting.Property.NodeScope);
 
+    public static final String ELASTIC_INFERENCE_SERVICE_SSL_CONFIGURATION_PREFIX = "xpack.inference.elastic.http.ssl.";
```

Comment:
Are we sure we want to use an `http` namespace here?

Reply:
Yes, I'm sure. It's being used only in HTTP in our case (see [xpack.security.http.ssl](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-settings.html#http-tls-ssl-settings) as a ref). 
TLS can encode different types of traffic like FTP, SMTP, VPN, etc., so I think it makes sense to use the `http` prefix.